.github/workflows/cd-controls.yml

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) Files Community
+# Licensed under the MIT License.
+
+# Abstract:
+#  Deploys the Files UI Controls library to NuGet
+#
+# Workflow:
+#  1. Restore and build Files UI Controls
+#  2. Generate a NuGet package and symbols
+#  3. Publish the artifacts to NuGet
+
+name: Files CD (UI Controls)
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
This workflow should automatically run after bumping the version number. However, it should not run every time we merge a PR into `main`, otherwise it will create extra noise which we wish to avoid.

Reply:
Can't we test by bumping the version number?

Reply:
I don't understand

Reply:
I meant we can test by modifying `CurrentVersion.props`

Reply:
Yes, but it's nicer to test here by just any commit.
Regardless it won't work because there isn't a token yet.
It doesn't matter at the moment.

Reply:
The token is already configured, if everything else is ready, we should make this change and merge.